packages/react/src/components/FEaaS/BYOCWrapper.tsx

```diff
@@ -59,11 +59,9 @@ export class BYOCComponent extends React.Component<BYOCComponentProps> {
       const noNameProps = {
         errorOverride: 'BYOC: The ComponentName for this rendering is missing',
       };
-      return props.missingComponentComponent ? (
-        <this.props.missingComponentComponent {...noNameProps} />
-      ) : (
-        <MissingComponent {...noNameProps} />
-      );
+      const MissingComp = this.props.missingComponentComponent;
+
+      return MissingComp ? <MissingComp {...noNameProps} /> : <MissingComponent {...noNameProps} />;
```

Comment:
Do we need to apply these changes? I'm asking because in the same component we have two extra similar cases

Reply:
We get an error here if i wont rearrange the code. This is not a functional change.